File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all/c/src-c/main/java/api/BazService.java
package api;

import akka.stream.javadsl.Source;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import com.lightbend.lagom.javadsl.api.Descriptor;
import com.lightbend.lagom.javadsl.api.Service;
import com.lightbend.lagom.javadsl.api.transport.Method;
import static com.lightbend.lagom.javadsl.api.Service.*;

public interface BazService extends Service {

  ServiceCall<NotUsed, String> baz();

  @Override
  default Descriptor descriptor() {
    return named("/c").withCalls(restCall(Method.GET,  "/baz",    this::baz)).withAutoAcl(true);
  }
}

File: dev/sbt-plugin/src/sbt-test/sbt-plugin-commands/run-all-scaladsl/c/src-c/main/scala/api/BazService.scala
package api

import akka.NotUsed
import com.lightbend.lagom.scaladsl.api._
import com.lightbend.lagom.scaladsl.api.transport.Method

trait BazService extends Service {

  def baz: ServiceCall[NotUsed, String]

  override def descriptor = {
    import Service._
    named("c").withCalls(
      restCall(Method.GET, "/baz", baz)
    ).withAutoAcl(true)
  }
}
